hbapi/settings/heroku.py

```diff
@@ -14,3 +16,10 @@
 STATIC_URL = '/static/'
 
 ALLOWED_HOSTS = [".herokuapp.com", ]
+
+redis_url = urlparse.urlparse(os.environ.get('REDIS_URL'))
+REDIS_POOL = redis.ConnectionPool(host=redis_url.hostname,
```

Comment:
we're not using `REDIS_POOL` as constant, is it better to define it as local variable?